lib/web/terminal.go

```diff
@@ -1119,6 +1176,18 @@ func (t *TerminalStream) Close() error {
 	return trace.NewAggregate(t.ws.WriteMessage(websocket.BinaryMessage, envelopeBytes), t.ws.Close())
 }
 
+// Close send a close message on the web socket
+// prior to closing the web socket altogether.
+func (t *TerminalStream) Close() error {
+	if t.resizeC != nil {
+		t.once.Do(func() {
+			close(t.resizeC)
+		})
+	}
+
+	return t.WsStream.Close()
```

Comment:
Does this need to be pulled within the once or is that handled automatically inside of the WsStream close impl?